x/clp/keeper/migrations.go

```diff
@@ -13,6 +17,30 @@ func NewMigrator(keeper Keeper) Migrator {
 	return Migrator{keeper: keeper}
 }
 
+func (m Migrator) MigrateDenomToVer3(ctx sdk.Context) error {
+	pools := m.keeper.GetPools(ctx)
+	migrationMap := tkrKeeper.GetDenomMigrationMap()
+	for _, pool := range pools {
+		if peggy2Denom, found := migrationMap[pool.ExternalAsset.Symbol]; found {
+			peggy2Pool := *pool
+			peggy2Pool.ExternalAsset = &types.Asset{Symbol: peggy2Denom}
+			err := m.keeper.DestroyPool(ctx, pool.ExternalAsset.Symbol)
+			if err != nil {
+				return err
+			}
+			err = m.keeper.SetPool(ctx, &peggy2Pool)
+			if err != nil {
+				return err
+			}
+		} else {
+			return types.ErrMigrationFailed.Wrap(fmt.Sprintf("Unable to migrate pool : %s ", pool.String()))
+		}
+	}
+	return nil
+}
+
+// TODO : Add migration for Rewards after Branch is rebased with develop
+// TODO : Modify consensus version of CLP after Branch is rebased with develop
```

Comment:
@timlind @canercandan I'd like to know if we can remove both TODOs since there is no develop branch anymore.